rflx/model/message.py

```diff
@@ -93,6 +93,7 @@ class MessageState(Base):
     checksums: Mapping[ID, Sequence[expr.Expr]] = {}
 
 
+# pylint: disable=too-many-public-methods
 @invariant(lambda self: valid_message_field_types(self))
 @invariant(lambda self: not self.types if not self.structure else True)
 class AbstractMessage(type_.Type):
```

Comment:
This check shouldn't be disabled for the rest of the file.

Reply:
One day I will remember where to put those. Fixed in 1a980bf.